prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -180,7 +180,7 @@ public async Task<ActionResult> GetEnrolleeByUserId(Guid userId)
                 return Forbid();
             }
 
-            return RedirectToAction(nameof(GetEnrolleeById), new { enrolleeId = enrollee.Id });
+            return Redirect($"{PrimeConfiguration.Current.BackendUrl}/enrollees/{enrollee.Id}");
```

Comment:
I have confirmed in prod the env variable is:
BACKEND_URL=https://pharmanetenrolment.gov.bc.ca/api/v1/